### ISSUE TITLE
Upload to GQ Electronics/gmcmap not supplying microSieverts/hour values?

### ISSUE THREAD

**Issue by jult** [stelas/GCLog#4]
I've been using a GQ GMC 320+ V4 to upload data for about 2 months, the map shows:

CPM:25   ACPM:0   0uSv/h

Strangely it does update the CPM just fine. Not the ACPM and not uSv/h.
Is this related to the https redirects?

**Comment by stelas** [comment]
No, it has nothing to do with https since `http://www.GMCmap.com/log2.asp?AID=UserAccountID&GID=GeigerCounterID` `&CPM=nCPM` is used for data submission.

Currently the daemon only reads the CPMs as all devices and web services support it.

You're welcome to add this feature for GQ devices / gmcmap.com.
